src/main/java/uk/gov/pay/api/filter/ratelimit/RateLimiter.java

```diff
@@ -68,4 +70,21 @@ void auditRateOf(String key) {
             LOGGER.error("Unexpected error creating a Rate Limiter object in cache", e);
         }
     }
+
+    private int getRateForMethod(String method){
+        if(METHOD_POST.equals(method)){
+            return rateForPost;
+        }
+        return rate;
+    }
+
+    /**
+     * Returns rate to use local cache. It is the rate divided by no. of publicapi nodes
+     * 
+     * @param method GET, POST ...
+     * @return
+     */
+    private int getLocalRateForMethod(String method){
+       return getRateForMethod(method)/noOfNodes;
```

Comment:
what if this division doesn't divide nicely to an integer? does the rate round up or down to the nearest integer? be good to add tests for it

Reply:
Not applicable anymore. Removed noOfNodes and added new environment variables to use for local rate limiting